avaframe/ana3AIMEC/ana3AIMECCfg.ini

```diff
@@ -19,15 +12,29 @@ domainWidth = 600
 # angle for the start of the run-out zone
 startOfRunoutAreaAngle = 10
 
-# pressure limit value for evaluation of runout in [kPa]
-pressureLimit = 1
+# data result type for runout analysis (ppr, pfd, pfv)
+resType = ppr
+
+# limit value for evaluation of runout (depends on the resType chosen)
+thresholdValue = 1
+
+# contour levels value for the difference plot (depends on the resType chosen)
+# use | delimiter (for ppr 1|3|5|10, for pfd 0.1|0.25|0.5|0.75|1)
+contourLevels = 1|3|5|10
+
+# max of resType difference for contour lines plus capped difference in resType plot (for example 1 (pfd), 5 (ppr))
+diffLim = 1
 
 # chose interpolation method between 'nearest' and 'bilinear'
 interpMethod = bilinear
 
 # resampling step [m]
 distance = 10
 
+# computational module used for analysing the results of
```

Comment:
Can we write this in a clearer way? Especially in comparison to `comModules` below? It is not clear what the difference is. Maybe something like: 'which computational module is analysed` or `Target module for the analysis`....

Reply:
done